rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/health/10_basic.yml

```diff
@@ -8,6 +8,67 @@
       health_report: { }
 
   - is_true: cluster_name
-  - match:   { status: "green" }
-  - match:   { indicators.master_is_stable.status: "green" }
-  - match:   { indicators.master_is_stable.symptom: "The cluster has a stable master node" }
+  # This test might execute before the health node has received all health info, resulting in status "unknown"
+  - is_true: status
```

Comment:
Nit: I would suggest adding this in all tests because it's the overall status that should be present and can differentiate the requests we do from a single indicator request.